interface/billing/billing_process.php

```diff
@@ -2,34 +2,25 @@
 /*
  * Billing process Program
  *
- * This program process data for claims generation
- *
- * Copyright (C) 2016 Terry Hill <terry@lillysystems.com>
- * Copyright (C) 2014 Brady Miller <brady.g.miller@gmail.com>
- * Copyright (C) 2017 Jerry Padgett <sjpadgett@gmail.com>
- *
- * LICENSE: This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public License
- * as published by the Free Software Foundation; either version 3
- * of the License, or (at your option) any later version.
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- * GNU General Public License for more details.
- * You should have received a copy of the GNU General Public License
- * along with this program. If not, see http://opensource.org/licenses/gpl-license.php.
+ * This program processes data for claims generation
  *
  * @package OpenEMR
- * @author Terry Hill <terry@lilysystems.com>
  * @author Brady Miller <brady.g.miller@gmail.com>
+ * @author Terry Hill <terry@lilysystems.com>
  * @author Jerry Padgett <sjpadgett@gmail.com>
- * @link http://www.open-emr.org
+ * @author Stephen Waite <stephen.waite@cmsvt.com>
+ * @copyright Copyright (c) 2014 Brady Miller <brady.g.miller@gmail.com>
+ * @copyright Copyright (c) 2016 Terry Hill <terry@lillysystems.com>
+ * @copyright Copyright (C) 2017 Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2018 Stephen Waite <stephen.waite@cmsvt.com>
+ * @link https://github.com/openemr/openemr/tree/master
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
+ 
 require_once("../globals.php");
 require_once("$srcdir/patient.inc");
 require_once("$srcdir/billrep.inc");
 require_once("$srcdir/billing.inc");
-require_once("$srcdir/gen_x12_837.inc.php");
 require_once("$srcdir/gen_hcfa_1500.inc.php");
 
```

Comment:
Place following line of code here:
```
use OpenEMR\Billing\X12837P;
```

Reply:
btw, make sure there is an empty line before and after the above `use` line

Reply:
it's funny that psr 0 is able to use less lines due to placement in library/classes that are autoloaded but i guess it can be hidden from sight so that someone would have to go digging to find the class whereas the use lets you know kind of where it lives?